scipy/io/arff/arffread.py

```diff
@@ -27,10 +27,6 @@
 #    is lost!
 #   - Replace ValueError by ParseError or something
 
-# We know can handle the following:
-#   - numeric and nominal attributes
-#   - missing values for numeric attributes
-
```

Comment:
This was no longer current. It's redundant with the method docstring anyway; deleted so there's fewer places to update when things change.